frame/support/src/dispatch.rs

```diff
@@ -25,18 +25,21 @@ pub use frame_metadata::{
 };
 pub use crate::weights::{
 	SimpleDispatchInfo, GetDispatchInfo, DispatchInfo, WeighData, ClassifyDispatch,
-	TransactionPriority, Weight, PaysFee,
+	TransactionPriority, Weight, PaysFee, PostDispatchInfo,
 };
-pub use sp_runtime::{traits::Dispatchable, DispatchError, DispatchResult};
+pub use sp_runtime::{traits::Dispatchable, DispatchError};
 pub use crate::traits::{CallMetadata, GetCallMetadata, GetCallName};
 
+pub type DispatchResult = sp_runtime::DispatchResult<PostDispatchInfo>;
```

Comment:
This is a very personal nit but in such cases I prefer not using `use` and directly naming the crate so when reading it is implied immediately that the generic type is being fulfilled by something from the `weights` module. So something like 

```
pub type DispatchResult = sp_runtime::DispatchResult<weights::PostDispatchInfo>;
```

Nonetheless, you should document both types as they are public.

Reply:
The `use PostDispatchInfo` is mostly a re-export for other modules. I feel reluctant removing it for consistency reasons. However, I can still use the `weights` prefix in the type alias if that would make you happy.